examples/app/pivot/downloads/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
I have this gitignore file because users are able to download csv files to this folder.